LFPy/cell.py

```diff
@@ -680,29 +659,18 @@ def set_point_process(self, idx, pptype, record_current=False,
                             raise Exception(ERRMSG)
                     self.stimlist.append(stim)
 
-                    #record current
+                    # record current
                     if record_current:
-                        stimirec = neuron.h.Vector(int(self.tstop /
-                                                       self.dt+1))
-                        stimirec.record(stim._ref_i, self.dt)
-                        self.stimireclist.append(stimirec)
-                    else:
-                        stimirec = neuron.h.Vector(0)
-                        self.stimireclist.append(stimirec)
-
+                        self._stimitorecord.append(stim)
```

Comment:
these recorders are initialised in the `simulate` function based on the `variable_dt`